SUMMARY.md

```diff
@@ -157,6 +157,11 @@
 
 ---
 
+* [Advanced Routing](handout/routing-advanced/README.md)
```

Comment:
I don't think we need to explicitly call this out as 'Advanced Routing', I think renaming it from 'Route Guards' to just 'Lifecycle Hooks' and having it fall under Child Routes as 13.1.8 is fine